### DISCUSSION TITLE
Can't search for empty field

### DISCUSSION THREAD

**Discussion by sol1james** [netbox-community/netbox#9224] [Q&A] [answered]
Hi, I want to search for records that have no value in a field as per [this feature](https://github.com/netbox-community/netbox/issues/6138).  The url I'm using is:

```
https://netbox.mydomain/ipam/ip-addresses/?dns_name__empty=False
```

When I go to that URL I get all IP addresses, regardless of DNS name value.  The server is running Netbox 3.2.1.  Am I doing something wrong?

**Comment by jeremystretch** [comment, marked as answer]
Looks like a bug. I see that `dns_name__empty` is present in the filter set for IP addresses, however it's a MultiValueCharFilter rather than a BooleanFilter.

**Reply by sol1james** [reply]
I created an issue for this - https://github.com/netbox-community/netbox/issues/9231
